yarn-project/simulator/src/public/avm/opcodes/ec_add.ts

```diff
@@ -78,14 +78,13 @@ export class EcAdd extends Instruction {
       throw new EcAddPointNotOnCurveError(/*pointIndex=*/ 2, p2);
     }
 
-    const grumpkin = new Grumpkin();
     let dest;
     if (p1IsInfinite) {
       dest = p2;
     } else if (p2IsInfinite) {
       dest = p1;
     } else {
```

Comment:
I almost went and made Grumpkin and other curve types all have the same interface and in bb.js but walked it back as yarn-project mostly just implements things as they see fit. at least we can stop creating a do-nothing class instance